src/sdk/pynni/nni/compression/tensorflow/compressor.py

```diff
@@ -84,11 +74,12 @@ def select_config(self, layer):
             not be compressed
         """
         ret = None
+        if layer.type is None:
+            return None
         for config in self.config_list:
-            op_types = config.get('op_types')
-            if op_types == 'default':
-                op_types = default_layers.op_weight_index.keys()
-            if op_types and layer.type not in op_types:
+            config = config.copy()
```

Comment:
pytorch version also needs to do this copy

Reply:
Thanks, fixed.